tap_bing_ads/__init__.py

```diff
@@ -104,13 +104,16 @@ def create_sdk_client(service, account_id):
     LOGGER.info('Creating SOAP client with OAuth refresh credentials for service: %s, account_id %s',
                 service, account_id)
 
-    require_live_connect = CONFIG.get('require_live_connect', 'True') == 'True'
+    if CONFIG.get('require_live_connect', 'True') == 'True':
+        oauth_scope = BINGADS_MANAGE
+    else:
+        oauth_scope = MSADS_MANAGE
```

Comment:
Why there has to be a change to __init__.py for bingads migration to new version? Does that change how the oAuth is working?

Reply:
The BingAds SDK no longer accepts the `require_live_connect` parameter and you have to instead pass it the specific oauth_scope you want. The `bingads.manage` scope is the same as what it was using before.